docs/api.rst

```diff
@@ -6912,6 +6912,99 @@ Getting Compound balances
    :statuscode 500: Internal rotki error.
    :statuscode 502: An external service used in the query such as etherscan or the graph node could not be reached or returned unexpected response.
 
+Getting compound statistics
+=============================
+
+.. http:get:: /api/(version)/blockchains/ETH/modules/compound/stats
+
+   .. note::
+      This endpoint is only available for premium users
+
+   .. note::
+      This endpoint can also be queried asynchronously by using ``"async_query": true``
+
+   .. note::
+      This endpoint also accepts parameters as query arguments.
+
+   Doing a GET on the compound stats resource will return information related profit and loss of the events present in the database.
+
+
+   **Example Request**:
+
+   .. http:example:: curl wget httpie python-requests
+
+      GET /api/1/blockchains/ETH/modules/compound/stats HTTP/1.1
+      Host: localhost:5042
+
+   :reqjson bool async_query: Boolean denoting whether this is an asynchronous query or not
+   :reqjson int from_timestamp: Timestamp from which to query compound historical data. If not given 0 is implied.
+   :reqjson int to_timestamp: Timestamp until which to query compound historical data. If not given current timestamp is implied.
+
+   **Example Response**:
+
+   .. sourcecode:: http
+
+      HTTP/1.1 200 OK
+      Content-Type: application/json
+
+      {
+          "result": {
+              "interest_profit": {
+                  "0xA0B6B7fEa3a3ce3b9e6512c0c5A157a385e81056": {
+                      "eip155:1/erc20:0xc00e94Cb662C3520282E6f5717214004A7f26888": {
+                              "amount": "3.5",
+                              "usd_value": "892.5",
```

Comment:
This is not valid json. Please run it via a json validator. Trailing commas are not valid json and perhaps more.